Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,18 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
```

Comment:
the card css only has to be imported into one container, in this case this one (since it renders the others)